sdk/eventgrid/microsoft-azure-eventgrid/pom.xml

```diff
@@ -14,7 +14,13 @@
     <name>Microsoft Azure SDK for eventgrid</name>
     <description>This package contains Microsoft Azure EventGrid SDK.</description>
     <url>https://github.com/Azure/azure-sdk-for-java</url>
-    <licenses>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-data-sdk-parent</artifactId>
+        <version>1.0.0</version>
+        <relativePath>../../../pom.data.xml</relativePath>
+    </parent>
```

Comment:
Now that #4289 is filed, I would not make this change.

Reply:
As per my comment on the other PR - I think this change was required to make sure that the code actually built at all. Without this I would be lifting dependencies up into the POM which I think we are trying to avoid right now?

Reply:
Given this context - do you still want this change?

Reply:
I would be uncomfortable bringing in an inheritance hierarchy into a library that we haven't got any experience in. We could be introducing unexpected changes - different build rules, validation settings, etc. If this code isn't building before the move, I'm not sure we need to have it building after the move (immediately - it should definitely be followed up in #4289)?

Reply:
I've yanked the POM parent out - just waiting for the azure-sdk-for-java - client build to come green it looks like it got a few 500 errors from sonatype which caused the build to hiccup. Hopefully the issue is resolved.